bin/trivy-scan.sh

```diff
@@ -9,9 +9,10 @@ scan_target="$1"
 set +exo pipefail
 
 trivy \
+  --cache-dir /tmp/workspace/trivy-cache \
+  image \
```

Comment:
is this line intentional?

Reply:
Yeah. See https://github.com/aquasecurity/trivy/discussions/1515